### PR TITLE
Fix UX issues on archived experiment runs table

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
JIRA: https://issues.redhat.com/browse/RHOAIENG-8525

## Description

Fix UX issues:

1. Label spacing on breadcrumb

Screenshot 2024-06-13 at 10 07 27 AM

2. Kebab action menu overflow issue

Screenshot 2024-06-13 at 10 07 44 AM

## How Has This Been Tested?

Archive an experiment and go to the experiment runs table to check the UX fixes

## Test Impact

N/A

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by dpanshug — APPROVED** [review]
/lgtm

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Kebab action menu looks great - tested locally and the overflow issue is fixed 👍 

For the label I'm seeing some different spacing applied depending on the length of the experiment name: 
Screenshot 2024-06-14 at 1 53 00 PM
Screenshot 2024-06-14 at 2 00 27 PM

Wondering if we can update this to apply the same value regardless of the text length. What do you think @DaoDaoNoCode ?

**Comment by DaoDaoNoCode** [comment]
@jenny-s51 Yeah, that's a good finding, I will update that as well as the tests, thanks!

**Comment by DaoDaoNoCode** [comment]
@jenny-s51 Hi, do you know if we could remove the min-width of the truncated text in PF? There is always a min width, that's why there is a big space between the name and the label when the experiment name is short.

**Review by jenny-s51 — COMMENTED** [review]
@DaoDaoNoCode good question! Been investigating this.. the `minWidth` value is hardcoded in PF `Truncate` - so we could instead conditionally render `<Truncate>` if the `experiment?.display_name.length` exceeds a certain number of characters, and if it's shorter we would just render the `<Text>` component (since it would be expected that the truncation would not be applied there). 

What are your thoughts @DaoDaoNoCode ?

File: frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRow.tsx [hunk 1]
```diff
@@ -161,7 +161,11 @@ const PipelineRunTableRow: React.FC<PipelineRunTableRowProps> = ({
       {customCells}
       {hasRowActions && (
         <Td isActionCell dataLabel="Kebab">
-          <ActionsColumn items={actions} />
+          <ActionsColumn
+            data-testid="pipeline-run-table-row-actions"
+            items={actions}
+            popperProps={{ appendTo: () => document.body, position: 'right' }}
```
Comment (Gkrumbach07) [inline frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRow.tsx:167]: if you append to the body it gives different text styling compared to you appending it to the dashboard root

Comment (DaoDaoNoCode) [reply frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRow.tsx]: Updated, thanks.

**Comment by Gkrumbach07** [comment]
/lgtm
/approved

dropdown has the same styling now, good to merge

**Pull request merged**